cdap-sentry/cdap-sentry-extension/cdap-sentry-binding/src/test/java/co/cask/cdap/security/authorization/sentry/binding/SentryAuthorizerTest.java

```diff
@@ -100,6 +102,32 @@ public Properties getExtensionProperties() {
       }
 
       @Override
+      public void createTopic(String s) throws TopicAlreadyExistsException, IOException {
```

Comment:
It is not a TMS question. It is the authorization API issue. See my comment in CDAP-7994.